### ISSUE TITLE
RN 0.60+ support and Android X

### ISSUE THREAD

**Issue by quachsimon** [blackuy/react-native-twilio-video-webrtc#225]
We are looking to update our project to the latest version of RN (v0.65) and going through the docs I could not find anything on updating this Twilio lib.

All I know is there are breaking changes with 0.60 in regards to Android X and I was wondering if this is in the pipeline somewhere that may be addressed?

Has anyone run into any issues with v0.60+?

Thanks

**Comment by slycoder** [comment]
It's definitely on my radar but I haven't gotten around to it.  I was hoping that jettifier would magically take care of everything; has anyone tried with RN 0.60?  Let us know if there's anything obvious that needs to be done.

**Comment by azhararmar** [comment]
I am using it on RN 0.60.5 and for now it seems to work good, RN 0.6 autolinking seems to be working good as well.

**Comment by quachsimon** [comment]
@slycoder, is [Jetifier](https://github.com/mikehardy/jetifier#module-maintainers) already part of the current master branch (I would assume i'll need to update to the latest version to get it if so?)

**Comment by slycoder** [comment]
Not that I'm aware of --- but I was under the impression that as long as the parent project had jettifier set up it would work.  Feel free to disavow me of that =).

**Comment by azhararmar** [comment]
@quachsimon yes, I did get it to work for Android as well, downgrading the gradle version worked for me

```
distributionUrl=https\://services.gradle.org/distributions/gradle-5.3.1-all.zip
com.android.tools.build:gradle:3.4.1
```

**Comment by shamilovtim** [comment]
i don't see what gradle version has to do with it. it's complaining that a react bridge method doesn't exist. I've tried gradle 5.3.1 + android gradle 3.4.1 and it still throws error.

not only that but because i use expo i need it to work with 3.3.2 + 4.10.2

**Comment by shamilovtim** [comment]
https://github.com/react-native-community/discussions-and-proposals/issues/129

> Jetifier will not work for React Native projects as it only works on "packaged artifacts". Almost all React Native dependencies come in the form of a reference to the library source code. These are not migrated by Jetifier.

> The "best" solution is for all libraries to release a version which has migrated to AndroidX and the app developer can then upgrade all of their dependencies at once. This would mean, however, that users who want to get bug fixes for that library would need to migrate their app to AndroidX. This, in turn, means that if just one of their dependencies is not migrated to AndroidX, they are locked into using <=RN 0.59 and cannot use new versions of libraries which have migrated.

**Comment by shamilovtim** [comment]
got it working using android gradle: 3.3.2, gradle 4.10.2.

`yarn add -D jetifier`
`yarn rm react-native-twilio-video-webrtc` 
`rm /node_modules/` 
`yarn install` 
`yarn add https://github.com/blackuy/react-native-twilio-video-webrtc`
`npx jetify`
